code-challenges.test.js

```diff
@@ -14,38 +15,135 @@
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
+describe("remainderArray", () => {
+  it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3", () => {
+   const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+   const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+      expect(remainderArray(hodgepodge1)).toEqual([2, 0, -1, 0])
+      expect(remainderArray(hodgepodge2)).toEqual([2, 1, -1])
+   })
+})
+
+// Test Result 
+// ● remainderArray › takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
+// ReferenceError: remainderArray is not defined
 
 
 // b) Create the function that makes the test pass.
+// Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
+// Pseudocode
+// create a function : remainderArray
+// input is an array of mixed data (numbers and strings)
+// use .filter to iterate through the array and return only typeof 'numbers' 
+// then use .map to iterate through the new array and get a return of an array of only the remainders of the numbers when divided by 3 = use modulo operator. 
+
+const remainderArray = (array) => {
+  let numArray = array.filter(value => typeof value === 'number')
+  return numArray.map(value => value % 3) 
+}
+
+
+//  Test Passed 
+//  PASS  ./code-challenges.test.js
+//  remainderArray
+//    ✓ takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3 (1 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
 
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+
+describe("sumsCubed", () => {
+  it("takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+    const cubeAndSum1 = [2, 3, 4]
+    const cubeAndSum2 = [0, 5, 10]
+    expect(sumsCubed(cubeAndSum1)).toEqual(99)
+      expect(sumsCubed(cubeAndSum2)).toEqual(1125)
+   })
+})
+
+// Test Result 
+// ● sumsCubed › takes in an array of numbers and returns the sum of all the numbers cubed.
+// ReferenceError: sumsCubed is not defined
 
 
 // b) Create the function that makes the test pass.
+// Pseudocode 
+// create a function : sumsCubed 
+// input is an array of numbers 
+// we must first iterate through the array using .map and return all numbers cubed (which translates to number**3) 
+// once we get these cubed values, we must add all these values using the built-in method of 
+// .reduce 
+
+
+const sumsCubed = (array) => {
+  let cubedArray = array.map(value => value**3)
+    return cubedArray.reduce((a, b) => a + b)
+}
```

Comment:
Excellent work in this section!  Logic is great and your pseudocode has a wonderful breakdown of how you approached the problem.  Awesome job!